docs/build/javascript/quick-start.md

```diff
@@ -0,0 +1,146 @@
+# Quickstart
+Learn the basics by setting up and interacting with the [Ceramic Javascript Client](./javascript/installation.md).
+
+!!! warning ""
+    **Want an even faster way to try Ceramic? Visit the [Playground Demo App](https://playground.ceramic.dev){:target="_blank"} to test the full stack of Ceramic components in the browser.
+
+## **1. Install the Client**
+
+Visit the [Ceramic Client](./javascript/installation.md) page for instructions on how to quickly install the Client.
+
+## **2. Create a stream**
+Creating streams is dependent on the StreamType. More details can be found in our [StreamTypes Overview](../streamtypes/overview.md). A basic sample using the [TileDocument StreamType](../streamtypes/tile-document/overview.md) can be found below.
+
+=== "Command"
+
+    ```JavaScript
+    import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+    const schemaId = 'k3y52l7qbv1fry1fp4s0nwdarh0vahusarpposgevy0pemiykymd2ord6swtharcw' // this is the IDX Basic Profile Schema.
+    
+    const content = {
+      name: 'Your Name',
+      emoji: '✌🏻',
+      description: 'Something about yourself'
+    }
+    const metadata = {
+      controllers: [ceramic.did.id],
+      family: 'Basic Profile',
+      schema: schemaId
```

Comment:
Do we really want to demonstrate the use of an IDX schema / definition as the first example here? Wouldn't it be more easy to just create an hello world stream?

Reply:
It would be easier, but I feel like it would create a less holistic experience. I think that this example is easily parsable & would also work well as a reference for anyone just starting out.

Reply:
But isn't IDX covered in the IDX specific quickstart?

Reply:
That's true, I think I got this confused in my head. For some reason, I thought that creating a stream was after querying a stream. Will update this example though.

Reply:
I now remember why I had it set up how it was. We cover creating a stream later in the documentation and the schema must be a commitId. So it just made more sense to not duplicate the content.